logprep/abc/getter.py

```diff
@@ -10,6 +10,12 @@
 
 yaml = YAML(typ="safe", pure=True)
 
+BLACKLIST_VARIABLE_NAMES = [
```

Comment:
Could you name this to `BLOCKLIST`?

Reply:
why? it behaves like a blacklist. so I called it blacklist :-)

Reply:
done